tests/utils/verification/volumeproperties.go

```diff
@@ -161,3 +163,52 @@ func GetAssociatedPolicyName(hostname string, volName string) (string, error) {
 	}
 	return op, err
 }
+
+// CheckVolumeInProcMounts - check if volume entry is present in docker host mount file
+// grep does the work. If entry isn't present, err is not nil (Exit status 1)
+func CheckVolumeInProcMounts(hostName, volName string) bool {
+	cmd := "grep -q " + volName + vm.MountFile
+	_, err := ssh.InvokeCommand(hostName, cmd)
+
+	if err != nil {
+		log.Printf("No entry for volume %s in %s.", volName, vm.MountFile)
+		return false
+	}
+
+	return true
+}
+
+// CheckRefCountInLogs - check the log record for expected refcount of the volume.
+// grep does the work. If refcount isn't the expected one, err is not nil (Exit status 1)
+func CheckRefCountInLogs(hostName, volName string, logLines, expectedRefCnt int) bool {
```

Comment:
`CheckRefCount`? .. as long as method document mentions the correct intention .. no need to append in method name.